schedule/yam/test_cases/migration_mu.yaml

```diff
@@ -0,0 +1,24 @@
+---
+name: migr_12sp5
+schedule:
+  - migration/version_switch_origin_system
+  - boot/boot_to_desktop
+  - migration/check_system_registration
+  - migration/record_disk_info
+  - migration/reboot_to_upgrade
+  - migration/version_switch_upgrade_target
+  - installation/bootloader
+  - installation/welcome
```

Comment:
we are supposed to try with AutoYaST here instead of introducing more needle-base modules.

Reply:
you want a migration via Autoyast? I didn't realize that. Please verify.

Reply:
here: https://openqa.suse.de/tests/13934183#details I tried Johnathan's ay profile for upgrate and it works.
But still, if the test fails and reruns, it needs registering. Otherwise it fails with: https://openqa.suse.de/tests/13933716#step/installation/4

Reply:
yes, with autoyast would be more stable, but we need two jobs connected with START_AFTER_TEST and the parent with PUBLISH variable to see this problem you mentioned

Reply:
I have set the START_AFTER_TEST in my MR. Here I wasn't sure if it will be able to locate a job running in a different job group.

Reply:
START_AFTER_TEST does work among different job groups as far as the test suite are for the same product.